user/monolith/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "monolith"
+pkgver = "2.8.3"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["pkgconf"]
+makedepends = [
+    "rust-std",
+    "cargo-auditable",
```

Comment:
`cargo-auditable` belongs as first in `hostmakedepends`, try cross-building with `./cbuild -a aarch64 pkg user/monolith` assuming non-aarch64 host, also sort `makedepends` alphabetically

Reply:
> try cross-building with ./cbuild -a aarch64 pkg user/monolith assuming non-aarch64 host

it got built successfully, should i do anything else or push?

Reply:
I meant testing that before moving `cargo-auditable` to `hostmakedepends` locally just to see what happened before if a cross-build was attempted. things cross-compiling isn't mandatory but the dependencies at least should be placed in the correct list anyway which (always first) testing cross-build can help with